services/app-api/src/emailer/templateHelpers.ts

```diff
@@ -160,9 +164,12 @@ const generateCMSReviewerEmailsForContract = (
         ]
```

Comment:
Not sure if it'll get flagged in a merge conflict but [logic was added](https://github.com/Enterprise-CMCS/managed-care-review/pull/2603/files#diff-989f1f3bbd590ba8998db651e954e5d0806141f0ebfd80c3d1fe510ac28e5352) to these helper functions to not add oact to emails for non risked based contracts in a recent PR